kong/plugins/session/access.lua

```diff
@@ -78,7 +84,7 @@ function _M.execute(conf)
 
   s:start()
 
-  set_consumer(consumer, credential)
+  authenticate(consumer, credential, groups)
```

Comment:
just above we're checking whether the consumer still exists, and destroy the session if it's not the case. Shouldn't this also be validated for the groups?

Reply:
then it probably needs a note in the docs. 📝

Reply:
Added to the readme and updated the comments.